Detail/ViewComponents/DeliveryHelpView.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+
+public class DeliveryHelpView: UIView {
+
+    // MARK: - Internal properties
+
+    private let deliveryHelpImage = #imageLiteral(resourceName: "deliveryHelp")
```

Comment:
Could we avoid using #imageLiterals? I think we discussed this but I could provide more feedback if needed.

Reply:
Sure. I just did it for the convenience of it. I'll change it.